src/Template/Bake/Model/table.ctp

```diff
@@ -13,34 +13,42 @@
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
 use Cake\Utility\Inflector;
+
+$annotations = [];
+foreach ($associations as $type => $assocs) {
+    foreach ($assocs as $assoc) {
+        $typeStr = Inflector::camelize($type);
+        $annotations[] = "@property \Cake\ORM\Association\\{$typeStr} \${$assoc['alias']}";
+    }
+}
+$annotations[] = "@method {$entity} get(\$primaryKey, \$options = [])";
```

Comment:
I have no idea on that. I was adding a `use` for those, but if FQN is the preferred way then I'm fine with that.